airflow/operators/google_api_to_s3_transfer.py

```diff
@@ -35,7 +35,12 @@ class GoogleApiToS3Transfer(GoogleApiToS3TransferOperator):
     """
     This class is deprecated.
     Please use:
-    `airflow.providers.amazon.aws.operators.google_api_to_s3_transfer.GoogleApiToS3TransferOperator`."""
+    `airflow.providers.amazon.aws.operators.google_api_to_s3_transfer.GoogleApiToS3TransferOperator`.
+
+    .. seealso::
```

Comment:
This should not be added to this class. You can skip all files from the airflow.contrib directory.

Reply:
But this is not under `airflow.contrib`. So why should it be skipped in this case?

Reply:
It's a deprecated class. We will transfer many classes to the airflow.providers package. We only have references here to maintain backward compatibility.

Reply:
Formerly the core code was maintained by the original creators - Airbnb. The code that was in the contrib package was supported by the community. The project was passed to the Apache community and currently the entire code is maintained by the community, so now the division has no justification, and it is only due to historical reasons.
https://airflow.readthedocs.io/en/stable/_api/index.html
We wanted to fix this, so we created a new airflow.providers package to organize these operators.
In the airflow.{operators,sensors} package, we only have a small set of core classes.